.travis.yml

```diff
@@ -48,16 +48,43 @@ matrix:
 
     # Debug Builds
     - os: linux
-      compiler: "gcc-6-debug-cov-asan"
+      compiler: "format-taginfo-docs"
+      env: NODE=6
+      sudo: false
+      before_install:
+      install:
+        - source $NVM_DIR/nvm.sh
+        - nvm install $NODE
+        - nvm use $NODE
+        - npm --version
+        - npm install --ignore-scripts
+        - npm link --ignore-scripts
```

Comment:
Not sure why the link step is needed?

Also the empty `before_install` above is dead code?